src/SchemaEditor.tsx

```diff
@@ -17,6 +21,7 @@ const editorPreamble = `
 `.trim();
 
 export const SchemaEditor: React.FC<SchemaEditorProps> = (props) => {
+  const { initialContent, schema, validationRange, onValidate } = props;
```

Comment:
Alignment seems off?

Reply:
This whole file was using 3 spaces for indentation for some reason, fixed in next PR